patches/minecraft/net/minecraft/block/state/IBlockProperties.java.patch

```diff
@@ -14,7 +14,15 @@
  
      @SideOnly(Side.CLIENT)
      boolean func_185895_e();
-@@ -90,5 +94,10 @@
+@@ -73,6 +77,7 @@
+ 
+     IBlockState func_185899_b(IBlockAccess p_185899_1_, BlockPos p_185899_2_);
+ 
++    @Deprecated // This is only a fallback method. Forge will now get the bounding box from RayTraceResult#boxHit
```

Comment:
Would probably recommend documenting the @Deprecated annotation according to http://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html, rather than code comments. More likely someone would see why they should not be using it.